### DISCUSSION TITLE
User.name problem

### DISCUSSION THREAD

**Discussion by amsaighi** [community/community#22621] [New to GitHub]
<p>Hello everyone, </p>
<p>I have issue when I’m trying commit to gitbut repository, they sane my user name and profisional mail in the commit, </p>
<p>I change with command but the same thing, the commit save my machine name </p>
<pre><code>git config --global user.name and user.email
</code></pre>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/d7b50a4c5317f79d6c0176ccbf155a39df132a4a.png" data-download-href="/uploads/short-url/uMex31nr0BlXfUap2OJRtuitvvk.png?dl=1" title="github.PNG" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/3142753/181072198-c9f468f4-dfef-4f96-be82-3a655f3198ef.png" alt="github.PNG" data-base62-sha1="uMex31nr0BlXfUap2OJRtuitvvk" width="690" height="191" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/d7b50a4c5317f79d6c0176ccbf155a39df132a4a_2_690x191.png, https://aws1.discourse-cdn.com/github/original/1X/d7b50a4c5317f79d6c0176ccbf155a39df132a4a.png 1.5x, https://aws1.discourse-cdn.com/github/original/1X/d7b50a4c5317f79d6c0176ccbf155a39df132a4a.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/d7b50a4c5317f79d6c0176ccbf155a39df132a4a_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">github.PNG</span><span class="informations">886×246 15.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>how can I related my vscode to any github repository 
<p>thank you  </p>

**Comment by mpboom** [comment]
<p>You need to run them as two seperate commands:</p>
<pre><code>git config --global user.name [YOUR FULL NAME]

git config --global user.email [YOUR MAIL ADDRESS, THAT IS ALSO LINKED TO YOUR GH ACCOUNT]
</code></pre>

**Comment by amsaighi** [comment]
<p>that’s what I did before posting this, the same result </p>

**Comment by mpboom** [comment]
<p>OK, and what exactly is the problem then? Is Git comitting with a different e-mailadress then you would like Git to commit with?</p>
<p>I see you mentioning VSCode in you post. Is the behaviour (the mail address in this case I believe) different based on if you use VSC or Git Bash?</p>

**Comment by amsaighi** [comment]
<p>Hello, </p>
<p>Do you know how to relate github to vscode ? </p>
<p>I tried to change my email and name, but when I commit they save my alias and mail not my username Github</p>
<p>thank you </p>

**Comment by amsaighi** [comment, marked as answer]
<p>solution is here </p>
<p><a href="https://marketplace.visualstudio.com/items?itemName=GitHub.vscode-pull-request-github" title="GitHub Pull Requests" rel="nofollow noopener">GitHub Pull Requests</a></p>
<p>good luck</p>
